### ISSUE TITLE
[win][v469a]OpenGL too dark in-game even at brightness set at 10/10

### ISSUE THREAD

**Issue by gargul2** [OldUnreal/UnrealTournamentPatches#45]
I've been always using OpenGL renderer in my UT 436 client, with a brightness set at 7/10,
now after update to 469a patch, OpenGL renderer has become useless, because it's just too dark,
setting it at 7/10 just like in 436 makes it looks like 4/10 in 436, and even when I set it at 10/10 it still is too dark like 5-6 in UT 436.

(There have been a similiar "too dark" issue always happening with Direct3D10 in UT 436 for me, althought in this case the brightness was completely stucked and not even responding to F11 key, just like in this reported case:
https://www.reddit.com/r/unrealtournament/comments/bw3a8a/ut_goty_brightness_does_nothing_too_dark_to_play/
the same happens with Direct3D10 in 469 for me as well.)

I've made screenshoots of the OpenGL issue on both 436 and 46  to provide a comparision between 436 and 469 brightness, but the screenshoots turned out to be useless, as they do not represent the same brightness like playing real game, then I've made photos using my smartphone but it turned out to be the same, so it's best if you stick to what I said at the beginning (about brigtness levels X/Y).

**Comment by SeriousBuggie** [comment]
Can confirm issue.   
On v436 OpenGL brightness range is fine.   
On v469a OpenGL brightness range too low. Look like start from same values as before, but end too dark.   
Same with XOpenGL on v469a - upper bound still dark enough.    
D3D9 fine on v436 and v469a.

**Comment by gargul2** [comment]
> Same with XOpenGL on v469a - upper bound still dark enough.

Yep, I have been aware, but after setting it at 10/10 it is equal to 7/10 in 436 so it fit my needs, that's why I felt no need to mention it, but yes, my needs are just my needs, so for others who use more than 7/10 in 436,  yes - XOpenGL on v469a is broken as well.

> I want note: brightness changed when drag slider, but not enough.

Yep, a step between the brigthness levels, is too small.

**Comment by SeriousBuggie** [comment]
http://www.kentie.net/article/d3d10drv/
D3D10v28 brigthness fine in both v436 and v469a.  
But in v469a system font look like a shit.

**Comment by gargul2** [comment]
Thx

>  But in v469a system font look like a shit.

Yep, I've noticed as well.

**Comment by CacoFFF** [comment]
You may use the old gamma ramp system by disabling `UseFragmentProgram` (which disables shader code)
Or you may manually input the gamma correction in the shader by changing the following to true: `UseShaderGamma,
ShaderGammaCorrectHUD` and then changing the corresponding `GammaOffset...` .

**Comment by gargul2** [comment]
I've added `UseFragmentProgram=False` to `[OpenGLDrv.OpenGLRenderDevice] ` in `UnrealTournament.ini` and it's
working! Thx. Now I don't need to bother with bugged XOpenGL anymore as for now.

**Comment by SeriousBuggie** [comment]
Same for me. `UseFragmentProgram=False` for OpenGL fix dark brightness for it.
